ds_judgements_public_ui/templates/includes/pagination.html

```diff
@@ -0,0 +1,74 @@
+<nav class="pagination" aria-label="Results pagination">
+  <h3>Results pagination</h3>
+  <ul class="pagination__list">
+    <li class="pagination__list-item">
+      <span class="pagination__page-chevron-previous--disabled">
+        <span>
+          Previous page
+        </span>
+      </span>
+    </li>
+    <li class="pagination__list-pages">
+      <ol>
+        <li class="pagination__list-item">
+          <a class="pagination__page-link-current" href="#" aria-label="Current page, Page 1"
+            aria-current="true"><span>Current page </span>1
+          </a>
+        </li>
+        <li class="pagination__list-item">
+          <a class="pagination__page-link" href="#" aria-label="Go to page 2">
+            <span>Page </span>2
+          </a>
+        </li>
+        <li class="pagination__list-item">
+          <a class="pagination__page-link" href="#" aria-label="Go to page 3">
+            <span>Page </span>3
+          </a>
+        </li>
+        <li class="pagination__list-item">
+          <a class="pagination__page-link" href="#" aria-label="Go to page 4">
+            <span>Page </span>4
+          </a>
+        </li>
+        <li class="pagination__list-item">
+          <a class="pagination__page-link" href="#" aria-label="Go to page 5">
+            <span>Page </span>5
+          </a>
+        </li>
+        <li class="pagination__list-item">
+          <a class="pagination__page-link" href="#" aria-label="Go to page 6">
+            <span>Page </span>6
+          </a>
+        </li>
+        <li class="pagination__list-item">
+          <a class="pagination__page-link" href="#" aria-label="Go to page 7">
+            <span>Page </span>7
+          </a>
+        </li>
+        <li class="pagination__list-item">
+          <a class="pagination__page-link" href="#" aria-label="Go to page 8">
+            <span>Page </span>8
+          </a>
+        </li>
+        <li class="pagination__list-item">
+          <a class="pagination__page-link" href="#" aria-label="Go to page 9">
+            <span>Page </span>9
+          </a>
+        </li>
+        <li class="pagination__list-item">
+          <a class="pagination__page-link" href="#" aria-label="Go to page 10">
+            <span>Page </span>10
+          </a>
+        </li>
+        <li class="pagination__list-item">
+          <a class="pagination__page-link" href="#" aria-label="Go to page 11">
+            <span>Page </span>11
+          </a>
+        </li>
+      </ol>
+    </li>
+    <li class="pagination__list-item">
+      <a class="pagination__page-chevron-next" href="#"><span>Next page</span></a>
+    </li>
+  </ul>
+</nav>
```

Comment:
Should this output be dynamic, based on what the API returns?

Reply:
Django has some built-in mechanisms for pagination, which we could use here (with a bit of bending). Happy to pair program some time next week if you'd like to explore?

Reply:
Yes, @ababic - you're right. It should be dynamic to reflect the current page. @LornaHarwood - when making this dynamic we need to ensure we're keeping the `aria-label` and `aria-current` properties and values represented in the static version.

Reply:
The dynamic things will be added in later, this PR is just bringing the template across to where we can start implementing that. @ababic the notes you've added here will be very helpful when we do that, thanks 👍🏻